docs/lint_errors.md

```diff
@@ -122,58 +122,61 @@ nf-core pipelines must have CI testing with GitHub Actions.
 
 ### GitHub Actions
 
-There are 3 main GitHub Actions CI test files: `ci.yml`, `linting.yml` and `branch.yml` and they can all be found in the `.github/workflows/` directory.  
+There are 3 main GitHub Actions CI test files: `ci.yml`, `linting.yml` and `branch.yml` and they can all be found in the `.github/workflows/` directory.
```

Comment:
I thought about adding here a note that if these files are not present, the best option is to update to the newest version of the template so that these files are automatically created.

Reply:
I think that the tests will fail before we get here if the files are missing, so shouldn't be an issue I think..